lib/experimental/fonts/font-library/class-wp-rest-font-families-controller.php

```diff
@@ -44,363 +44,492 @@ public function register_routes() {
 	 */
-	public function uninstall_fonts( $request ) {
-		$fonts_to_uninstall = $request->get_param( 'font_families' );
-
-		$errors    = array();
-		$successes = array();
+	public function delete_item( $request ) {
+		$font_family_id = $request->get_param( 'id' );
+		$force          = isset( $request['force'] ) ? (bool) $request['force'] : false;
 
-		if ( empty( $fonts_to_uninstall ) ) {
-			$errors[] = new WP_Error(
-				'no_fonts_to_install',
-				__( 'No fonts to uninstall', 'gutenberg' )
-			);
-			$data     = array(
-				'successes' => $successes,
-				'errors'    => $errors,
+		// We don't support trashing for revisions.
+		if ( ! $force ) {
+			return new WP_Error(
+				'rest_trash_not_supported',
+				/* translators: %s: force=true */
+				sprintf( __( "Font faces do not support trashing. Set '%s' to delete.", 'gutenberg' ), 'force=true' ),
+				array( 'status' => 501 )
 			);
-			$response = rest_ensure_response( $data );
-			$response->set_status( 400 );
-			return $response;
 		}
 
-		foreach ( $fonts_to_uninstall as $font_data ) {
-			$font   = new WP_Font_Family( $font_data );
-			$result = $font->uninstall();
-			if ( is_wp_error( $result ) ) {
-				$errors[] = $result;
-			} else {
-				$successes[] = $result;
-			}
+		$deleted = parent::delete_item( $request );
+
+		if ( is_wp_error( $deleted ) ) {
+			return $deleted;
+		}
+
+		foreach ( $this->get_font_face_ids( $font_family_id ) as $font_face_id ) {
+			wp_delete_post( $font_face_id, true );
 		}
-		$data = array(
-			'successes' => $successes,
-			'errors'    => $errors,
-		);
-		return rest_ensure_response( $data );
 	}
 
 	/**
-	 * Checks whether the user has permissions to update the Font Library.
+	 * Prepares a single font family output for response.
 	 *
 	 * @since 6.5.0
 	 *
-	 * @return true|WP_Error True if the request has write access for the item, WP_Error object otherwise.
+	 * @param WP_Post         $item    Post object.
+	 * @param WP_REST_Request $request Request object.
+	 * @return WP_REST_Response Response object.
 	 */
-	public function update_font_library_permissions_check() {
-		if ( ! current_user_can( 'edit_theme_options' ) ) {
-			return new WP_Error(
-				'rest_cannot_update_font_library',
-				__( 'Sorry, you are not allowed to update the Font Library on this site.', 'gutenberg' ),
-				array(
-					'status' => rest_authorization_required_code(),
-				)
-			);
+	public function prepare_item_for_response( $item, $request ) { // phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable -- required by parent class
+		$data = array();
+
+		$data['id']                   = $item->ID;
+		$data['theme_json_version']   = 2;
+		$data['font_faces']           = $this->get_font_face_ids( $item->ID );
+		$data['font_family_settings'] = $this->get_settings_from_post( $item );
+
+		$response = rest_ensure_response( $data );
+		$links    = $this->prepare_links( $item );
+		$response->add_links( $links );
```

Comment:
```suggestion
if ( rest_is_field_included( '_links', $fields ) || rest_is_field_included( '_embedded', $fields ) ) {			
		$links    = $this->prepare_links( $item );
		$response->add_links( $links );
}		
```

Do not prepare and add links unless requested in _fields

Reply:
Done in https://github.com/WordPress/gutenberg/pull/57688/commits/32689e176c2d2df55803a1fa06903ac148e8df7d